### ISSUE TITLE
No way to specify window resolution

### ISSUE THREAD

**Issue by globalcitizen** [paulofmandown/rotLove#8]
If you call **ROT.Display()** it only accepts arguments in characters, ie. specifying that the console should be X by Y characters (of the supplied font).

This is annoying, for example in the case that you want to go full screen at some resolution or other then back-calculate text width and height resolution to make optimum use of the available screen space.

**Comment by paulofmandown** [comment]
I agree, that should be easier. There used to be a scale option in Display, but I appear to have removed it by accident some time ago. I've added it back in 833bdcd

I'd like to add a setResolution, and pull the bits that would have to be re-called out of __init and into a separate function.
